charts/rds-auth-proxy/templates/cert.yaml

```diff
@@ -0,0 +1,36 @@
+{{- if .Values.proxy.certManager.enabled }}
+apiVersion: cert-manager.io/v1alpha2
+kind: Certificate
+metadata:
+  name: {{ include "app.clientCertName" . }}
+spec:
+  dnsNames:
+    - {{ include "app.fullname" . }}.{{ .Release.Namespace }}.svc
+    - {{ include "app.fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local
```

Comment:
This assumes cluster domain name is default of `cluster.local`.

Reply:
what would the suggested resolution be?

Reply:
Add a `clusterName` to the values file and interpolate here.